woocommerce/woocommerce-android
# [TTP] Remove feature and beta feature flags

Closes: #9384


### Description

Removes both TTP flags
* Beta feature flag
* Feature flag

### Testing instructions

* Run all the Tap To Pay flows in the release
* Make sure that all of the work

More info - pdfdoF-3in-p2
